sources/osg/State.js

```diff
@@ -457,31 +468,25 @@ State.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( Objec
         }
     },
 
-    applyTextureAttribute: function ( unit, attribute ) {
+    applyTextureAttribute: function ( unit, attribute, force ) {
```

Comment:
why adding force ?
It looks like it's not understood or we are not what's happened.
I dont want to change api because of that